kcalcore/icaltimezones.cpp

```diff
@@ -1313,10 +1313,10 @@ QList<QDateTime> ICalTimeZoneSourcePrivate::parsePhase( icalcomponent *c,
   }
 
   // Convert DTSTART to QDateTime, and from local time to UTC
-  const QDateTime localStart = toQDateTime( dtstart );   // local time
   dtstart.second -= prevOffset;
   dtstart.is_utc = 1;
   const QDateTime utcStart = toQDateTime( icaltime_normalize( dtstart ) );   // UTC
+  const QDateTime localStart = utcStart.toLocalTime();  // local time
```

Comment:
I built an upgraded libical, but I can't install it without updating a bunch of other packages which require it:

	libical.so.0 is needed by (installed) nemo-qml-plugin-calendar-qt5-lightweight-0.1.4-1.armv7hl
	libical.so.0 is needed by (installed) nemo-qml-plugin-calendar-qt5-0.1.4-1.armv7hl
	libical.so.0 is needed by (installed) mkcal-qt5-0.3.27-1.16.1.armv7hl
	libical.so.0 is needed by (installed) kcalcore-qt5-4.10.2-1.armv7hl
	libicalss.so.0 is needed by (installed) nemo-qml-plugin-calendar-qt5-lightweight-0.1.4-1.armv7hl
	libicalss.so.0 is needed by (installed) nemo-qml-plugin-calendar-qt5-0.1.4-1.armv7hl
	libicalss.so.0 is needed by (installed) mkcal-qt5-0.3.27-1.16.1.armv7hl
	libicalss.so.0 is needed by (installed) kcalcore-qt5-4.10.2-1.armv7hl
	libicalvcal.so.0 is needed by (installed) nemo-qml-plugin-calendar-qt5-lightweight-0.1.4-1.armv7hl
	libicalvcal.so.0 is needed by (installed) nemo-qml-plugin-calendar-qt5-0.1.4-1.armv7hl
	libicalvcal.so.0 is needed by (installed) mkcal-qt5-0.3.27-1.16.1.armv7hl
	libicalvcal.so.0 is needed by (installed) kcalcore-qt5-4.10.2-1.armv7hl

Given that the second patch doesn't involve calls to libical, I don't think there's much point in going to that effort just yet...